src/app/static/src/app/components/plots/choropleth/MapAdjustScale.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+    <div class="p-3 ml-2">
+        <div class="form-check dynamic-filtered">
+            <label class="form-check-label">
+                <input id="type-input-dynamic-filtered" class="form-check-input" type="radio" :name="scaleTypeGroup"
+                       :value="scaleType.DynamicFiltered"
+                       v-model="selectedScaleType">
+                <span v-translate="'filteredDataset'"></span>
```

Comment:
all these are misaligned, i think the css property you want is `vertical-align: sub`, you can create a class for each of the spans here and add that css prop to the class

Reply:
Thanks, I've tried to improve the styling here a bit more generally
* Make width at either side more even
* Aligned the number inputs with the "Custom" label to hopefully make it more clear they belong to that
* Sorted out the vertical alignment as you suggest
* Made the number selection boxes wider to handle large numbers better